lib/citgm.js

```diff
@@ -29,14 +30,38 @@ exports.windows = windows;
  * 5. Output the results.
  **/
 
-function find(app, context, next) {
-  which(app, function(err, resolved) {
+function findNode(context, next) {
+  which('node', function(err, resolved) {
     if (err) {
-      next(Error(app + ' not found in path!'));
+      npmWhich(__dirname)('node', function(err, resolved) {
```

Comment:
What is the purpose of calling npmWhich here? If the node binary was not found by which how would this improve anything? I feel like I'm missing something here

Reply:
re your below comment, I don't think we want to be specifically supporting the use case of `node` being in the `node_modules` folder.

Open to other people's thoughts here

Reply:
Ah, good catch! I forgot to remove this when I moved to using the new lookup for npm and yarn